### DISCUSSION TITLE
How about using this for support tickets ?

### DISCUSSION THREAD

**Discussion by Benjozork** [flybywiresim/aircraft#2290] [Q&A] [answered]
🤔

**Comment by tareksabet** [comment, marked as answer]
testing

**Reply by OddSobriquet** [reply]
so here's my two cents on this.
I think the discussions feature is quite nice if you want to get more detailed information on a problem you have which you have looked a bit into already and started trouble shooting (very basic troubleshooting).
The help channel on discord on the other hand was great to verify a problem actually exists or if it's just one's own stupidity and lack of ability to work the mod correctly.
In other words. This discussions feature is good for: "Hey I'm facing issue AB which I can't seem to solve. I've already tried x,y,z and nothing worked."
The help channel was fantastic for: "Hey, I'm facing issue CD, anyone else?"

The inhibition (is that the correct word?) on just quickly asking whether an issue exists is much lower on discord compared to here (I see how this is an advantage for you), where you need to open up another ticket. Furthermore, forums (which this pretty much is) also have the problem to sidetrack and possibly have a tiny question result in a massive rat's tail discussion on absolutely everything.
Now, I understand that this is an attempt to illiminate the duplicate questions and I do agree it was getting and is necessary and I also understand that moderating the clusterf* the help channel was at times was more than just labour extensive but pretty exhausting.

I could imagine that maybe if a bot deleted messages in a discord help channel which are older than (just an example) let's say 1 or 2 hours (don't know if this is possible, either) it would still encourage to turn to this git forum for more serious trouble shooting while these "question to verify" maintain in the discord. What I would think is important though, is to still have people search this place first before asking on discord!

I hope this isn't too harshly worded, I'm not the best when it comes to that.

**Comment by tareksabet** [comment]
> 🤔

how can I close this ☺

**Comment by pepperoni505** [comment]
this is cool

**Reply by Snapmatics** [reply]
is it?

**Reply by Benjozork** [reply]
Yes

**Comment by Snapmatics** [comment]
Huh hey?

**Comment by ZigTag** [comment]
What is a32nx

**Comment by pepperoni505** [comment]
how do i make the JAVA code work?????

**Reply by Snapmatics** [reply]
By making HTML code work.

**Reply by ZigTag** [reply]
@Snapmatics I think you need to add the RPN

**Comment by ImenesFBW** [comment]
My plane do this.

![image](https://user-images.githubusercontent.com/71195324/101547247-42a0f380-39aa-11eb-9edb-a2e739418fb6.png)

**Reply by NathanInnes** [reply]
Update to latest master and should be like this
![image](https://user-images.githubusercontent.com/18389085/101667902-ceb52880-3a47-11eb-8268-f9c25ecd5e23.png)
